### PR TITLE
docs: fix Arch installation steps, and some typos

### PR DESCRIPTION
**Pull request by xTVaser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,11 +159,11 @@ nix-build -A packages.x86_64-linux.jak-asan # package with Clang ASan build
 
 ### Required Software
 
-We primarily use Visual Studio for development on Windows for C++ development.  Download the community addition from [here](https://visualstudio.microsoft.com/vs/)
+We primarily use Visual Studio on Windows for C++ development.  Download the community edition from [here](https://visualstudio.microsoft.com/vs/)
```
Comment (ManDude) [inline README.md:162]: might wanna specify 2022 perhaps?

Comment (xTVaser) [reply README.md:162]: I'll specify the latest release so we won't have to update the docs and just make sure it functions on the latest version.

**Comment by xTVaser** [comment]
Tested on WSL, i was able to fully build and run the game using the steps in the README (so fast as well)

**Review by water111 — APPROVED** [review]
thanks!

**Pull request merged**